app/models/ad.rb

```diff
@@ -19,7 +19,7 @@ class Ad < ApplicationRecord
 
   validates :city, presence: true
   validates :description, presence: true
-  validates :kind, presence: true
+  validates :kind, presence: true, inclusion: { in: KINDS }
```

Comment:
What's the error message user sees here?

Reply:
He should not see anything. It can only happen if he adds new option (manually). But he will still see nice message "Tip oglasa nije u list" (or very similar).